### ISSUE TITLE
404 images on the stability dashboard

### ISSUE THREAD

**Issue by sylvestre** [mozilla/clouseau#58]
On https://mozilla.github.io/stab-crashes/, some images are 404.

**Comment by sylvestre** [comment]
Depends on #51

**Comment by marco-c** [comment]
I'm now using basically the same procedure as clouseau to get the top 50 crashes in the correlation tool, so #51 isn't strictly needed but it would make everything easier to maintain.

There was another issue, https://help.github.com/articles/files-that-start-with-an-underscore-are-missing/, which I fixed with https://github.com/mozilla/stab-crashes/commit/7a75500e86d52020981ba9e123f4184a54692c4a.

The remaining missing images are for signatures where no correlation was found. I've opened https://github.com/mozilla/stab-crashes/issues/6 to track the remaining issue.
